ChatQnA/kubernetes/intel/cpu/xeon/manifest/chatqna.yaml

```diff
@@ -779,7 +779,8 @@ spec:
             {}
       volumes:
         - name: model-volume
-          emptyDir: {}
+          persistentVolumeClaim:
```

Comment:
I think we need to change the test scripts too to create/delete the shared storage too to make CI happy and clean